src/config.ts

```diff
@@ -1,7 +1,7 @@
 import { MessageEmbed } from 'discord.js';
 
 export default {
-  BIO: [ 'description', 'github', 'twitter', 'youtube', 'instagram', 'linkedin' ],
+  BIO: ['description', 'github', 'twitter', 'youtube', 'instagram', 'linkedin'],
```

Comment:
This change is not required. 

```suggestion
  BIO: [ 'description', 'github', 'twitter', 'youtube', 'instagram', 'linkedin' ],
```

Reply:
Alright will undo it, its due to prettier extension I think